src/aura/BGE_DonationSelector/BGE_DonationSelector.cmp

```diff
@@ -116,20 +116,48 @@
+                                            isTrue="{!and(equals(donation.Id, v.selectedDonation.Id), !v.selectedDonation.applyPayment)}">
+                                        <lightning:icon iconName="utility:check" size="small"
+                                                        class="slds-p-right_small selectedCheckmark"
+                                                        alternativeText="{!$Label.c.stgCheckboxTrue}"
+                                                        title="{!$Label.c.stgCheckboxTrue}"/>
+                                        <lightning:formattedText
+                                                value="{!$Label.c.bgeFormSelectedOpportunity}"/>
+                                        <aura:set attribute="else">
+                                            <lightning:button
+                                                    value="{!donation}"
+                                                    label="{!$Label.c.bgeFormUpdateOpportunity}"
+                                                    onclick="{!c.handleDonationSelection}"
+                                                    variant="base"/>
+                                        </aura:set>
+                                    </aura:if>
+                                </lightning:layoutItem>
+                                <lightning:layoutItem size="12" padding="around_xx-small"
+                                                      class="slds-border_top slds-p-top_x-small slds-m-top_x-small">
+                                    <aura:if isTrue="{!and(equals(donation.Id, v.selectedDonation.Id), v.selectedDonation.applyPayment)}">
+                                        <lightning:icon iconName="utility:check" size="small"
+                                                        class="slds-p-right_small selectedCheckmark"
+                                                        alternativeText="{!$Label.c.stgCheckboxTrue}"
+                                                        title="{!$Label.c.stgCheckboxTrue}"/>
+                                        <lightning:formattedText
+                                                value="{!$Label.c.bgeFormSelectedOpportunityApplyNewPayment}"/>
+                                        <aura:set attribute="else">
+                                            <lightning:button
+                                                    value="{!donation}"
+                                                    label="{!$Label.c.bdiMatchedApplyNewPayment}"
+                                                    onclick="{!c.handleApplyPaymentSelection}"
+                                                    variant="base"/>
+                                        </aura:set>
+                                    </aura:if>
+                                </lightning:layoutItem>
+                            </lightning:layout>
+
+                            <!--<br/>-->
```

Comment:
Let's kill this line :)

Reply:
Ok - removed! Should show up after next push